iroha/config.json

```diff
@@ -23,5 +23,6 @@
   "PRIVATE_KEY": {
     "digest_function": "ed25519",
     "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
-  }
+  },
+  "ROOT_PUBLIC_KEY": "ed207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
```

Comment:
maybe we can put it under some sort of configuration?
"STATE_CONFIG" or something like this?

Reply:
If we move this logic into `init.rs` maybe it will be `INIT_CONFIG`?

Reply:
looks good to me - we can rename it later if needed